live-usb-creator/install_scripts/0_post_install_nochroot

```diff
@@ -58,12 +58,20 @@ cp /vagrant/syslinux-splash.png /mnt/sysimage/usr/share/anaconda/boot/syslinux-s
 ###############################################################################
 
 mkdir -p /mnt/sysimage/data/app/subzero
-cp -r /vagrant/data_app_subzero/* /mnt/sysimage/data/app/subzero
-# pre-compiled protobuf compiler
-cp /vagrant/protoc-3.14.0-linux-x86_64.zip /mnt/sysimage/data/app/subzero
-# python wheels for protobuf
-cp /vagrant/protobuf-3.14.0-cp36-cp36m-manylinux1_x86_64.whl /mnt/sysimage/data/app/subzero
-cp /vagrant/six-1.15.0-py2.py3-none-any.whl /mnt/sysimage/data/app/subzero
-cp /vagrant/Pillow-8.3.2-cp36-cp36m-manylinux_2_5_x86_64.manylinux1_x86_64.whl /mnt/sysimage/data/app/subzero
-# copy in a version of maven that works.
-cp /vagrant/apache-maven-3.8.4-bin.tar.gz /mnt/sysimage/data/apache-maven-3.8.4-bin.tar.gz
+cp /vagrant/data_app_subzero/subzero-cli.sh /mnt/sysimage/data/app/subzero/subzero-cli.sh
+
+# Copy development tools as well as dependencies only for the Dev flavored ISO
+if [ -f "/vagrant/live_scripts/.isotype_dev" ]; then
+  # pre-compiled protobuf compiler
+  cp /vagrant/protoc-3.14.0-linux-x86_64.zip /mnt/sysimage/data/app/subzero
+  # python wheels for protobuf
+  cp /vagrant/protobuf-3.14.0-cp36-cp36m-manylinux1_x86_64.whl /mnt/sysimage/data/app/subzero
+  cp /vagrant/six-1.15.0-py2.py3-none-any.whl /mnt/sysimage/data/app/subzero
+  cp /vagrant/Pillow-8.3.2-cp36-cp36m-manylinux_2_5_x86_64.manylinux1_x86_64.whl /mnt/sysimage/data/app/subzero
+  # copy in a version of maven that works.
+  cp /vagrant/apache-maven-3.8.4-bin.tar.gz /mnt/sysimage/data/apache-maven-3.8.4-bin.tar.gz
+  # custom sshd_config
+  cp /vagrant/data_app_subzero/sshd_config /mnt/sysimage/data/app/subzero
```

Comment:
Just curious, where was this copy statement before ? I see below in the `startup` script:
```cp /data/app/subzero/sshd_config /etc/ssh/sshd_config```